capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './decorator'
+
+# class capitalize_decorator
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.capitalize
+  end
```

Comment:
- [ ] this method should also return a capitalized correct_name method form of nameable
```
 def correct_name
      @nameable.correct_name.capitalize
    end
```